plugins/Terminal/js/utils/helpers.js

```diff
@@ -7,7 +7,7 @@ import url from 'url'
 import * as constants from '../constants/helper.js'
 
 export const checkSiaPath = () => new Promise((resolve, reject) => {
-	fs.stat(Path.join(SiaAPI.config.attr('siad').path, process.platform === 'win32' ? '../siac.exe' : '../siac'), (err) => {
+	fs.stat(Path.resolve(SiaAPI.config.attr('siac').path, './siac'), (err) => {
```

Comment:
Isn't `siac.path` the absolute path to siac?  As such, `Path.resolve(siacpath, ./siac)` would return [siacpath]/siac/siac, no?